portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/Search.testcase

```diff
@@ -408,15 +408,15 @@ definition {
 
 		Click(locator1 = "SearchResults#SEARCH_INSIGHTS_WIDGET_REQUEST_COPY_BUTTON");
 
-		SearchPortlets.viewCopiedContent(text = "explain");
+		SearchPortlets.viewCopiedContent(text = "\"explain\": true, \"highlight\":");
 
 		Search.openSearchPage();
 
 		SearchPortlets.searchEmbedded(searchTerm = "test");
 
 		Click(locator1 = "SearchResults#SEARCH_INSIGHTS_WIDGET_RESPONSE_COPY_BUTTON");
 
-		SearchPortlets.viewCopiedContent(text = "_shards");
+		SearchPortlets.viewCopiedContent(text = "\"_shards\": { \"total\": 1, \"failed\":");
```

Comment:
Change to
```
SearchPortlets.viewCopiedContent(text = "\"_shards\": { \"total\":);
```
Since the total value may change if OOTB assets are added in the future